examples/README.md

```diff
@@ -0,0 +1,151 @@
+
+### Pre-requisites
+
+-   You have access to the `spv-wallet` non-custodial wallet (running locally or remotely).
+-   [Taskfile](https://taskfile.dev/installation/) is installed on your environment.
+-   SPV Wallet go client instance is properly created and configured.
+
+> [!TIP]
+> To verify Taskfile installation run: `task` command in the terminal.
+
+```
+task: [default] task --list
+task: Available tasks for this project:
+* default:                                Display all available tasks.
+* fetch-user-contact-by-paymail:          Fetch user contact by given paymail.
+* fetch-user-contacts:                    Fetch user contacts page.
+* fetch-user-merkleroots:                 Fetch user Merkle roots page.
+* fetch-user-shared-config:               Fetch user shared configuration.
+* fetch-user-transaction:                 Fetch user transaction with a given ID.
+* fetch-user-transactions:                Fetch user transactions page.
+* fetch-user-utxos:                       Fetch user UTXOs page.
+* fetch-user-xpub:                        Fetch current authorized user's xpub info.
+* generate-keys:                          Generate keys for SPV Wallet API access.
+* user-contact-confirmation:              Confirm user contact with a given paymail address.
+* user-contact-remove:                    Remove user contact with a given paymail address.
+* user-contact-unconfirm:                 Unconfirm user contact with a given paymail address.
+* user-contact-upsert:                    Upsert user contact with a given paymail address.
+* user-draft-transaction:                 Create a user draft transaction.
+* user-invitation-accept:                 Accept user contact invitation with a given paymail address.
+* user-invitation-reject:                 Reject user contact invitation with a given paymail address.
+* user-transaction-metadata-update:       Update user transaction metadata with a given ID.
+* user-xpub-metadata:                     Update current authorized user's xpub metadata.
+```
+
+## Authroization 
+
+> [!CAUTION]
+> Don't use the keys which are already added to another wallet.
+
+
+> [!IMPORTANT] 
+> Additionally, to make it work properly, you should adjust the `ExamplePaymail` to align with your `domains` configuration in the `spv-wallet` instance.
+
+Before interacting with the SPV Wallet API, you must complete the authorization process.
+
+To begin, generate a pair of keys using the `task generate-keys command`, which is included in the dedicated Taskfile. 
+
+**Example output:**
+```
+==================================================================
+XPriv:  xprv1d77e47e-452c-453f-bc4c-a42748f8145f
+XPub:  xpubd82c277b-0a7e-482f-8ad8-e92958d15acb
+Mnemonic:  mnemonic
+==================================================================
+```
+
+Once the keys are successfully generated, you can use one of two methods to create an SPV Wallet Go Client instance. 
+
+1. _NewWithXPub_
+1. _NewWithXPriv_
```

Comment:
### minor
NewWithXPriv should rather be a first option as this is the suggested way.